apps/greencheck/management/commands/dump_green_domains.py

```diff
@@ -23,62 +36,141 @@ def get_conn_string(self):
         # change this if we ever leave mysql, obvs
         return f"mysql://{db['USER']}:{db['PASSWORD']}@{db['HOST']}/{db['NAME']}"
 
-    def export_to_sqlite(self, database_url: str, db_name: str):
-        return subprocess.run(
-            ["db-to-sqlite", database_url, db_name, "--table=greendomain"]
+    @classmethod
```

Comment:
Can you explain the reasoning for using the class and static methods here?

I can understand why you might use them, but then elsewhere in the `Command.handle()` in this management command the object is being instantiated and methods being called as if they regular old methods on an instance of the class.

It might work, but I found this confusing, as I couldn't understand what other value the `staticmethod` and  `@classmethod` decorators added.

I'm totally open to to using class/static methods, but it seem like you'd either want _not_ instantiate objects, and use them, or you'd keep instantiating objects, and then not rely on these decoration of the methods.

I'd prefer we use just one approach, as right now, when I read this I'm unsure if it's better to instantiate, or just call the methods on the class itself.

If I had to make a choice - because there is no real changing state, I think we could get away with not instantiating objects, but it's not something I feel strongly about.

Reply:
Most probably the `GreenDomainExporter` should be refactored because at the moment it is class "on paper" only but, in fact, shares no state among its methods and that's why they're decorated accordingly.

The instantiation (`exporter = GreenDomainExporter()`) is used to perform fewer modifications in the future if the methods become non-static class members.